@xen-orchestra/lite/src/locales/en.json

```diff
@@ -180,5 +180,6 @@
   "version": "Version",
   "vm-is-running": "The VM is running",
   "vms": "VMs",
+  "vm-count": "VMs running",
```

Comment:
Manages pluralization.
Also try to use a message ID that matches the content of the message. (eg: `vm-running`)